src/dashboard/src/apache/django.wsgi

```diff
@@ -25,7 +27,20 @@ if path not in sys.path:
 
 os.environ['DJANGO_SETTINGS_MODULE'] = 'settings.common'
 
-application = get_wsgi_application()
+# workaround to troubleshoot mod_wsgi issues
+# See http://stackoverflow.com/questions/30954398/django-populate-isnt-reentrant 
+
+try:
+    application = get_wsgi_application()
+    print 'WSGI without exception'
+except Exception:
+    print 'handling WSGI exception'
```

Comment:
Please update print statements to be compatible with Python3 eg `print('handling WSGI exception')`